source/data_at_rest_encryption.rst

```diff
@@ -17,9 +17,49 @@ files can be decrypted and read by parties with the decryption key.
 
 The |feature| in |PSMDB| is introduced in version 3.6 to be compatible with
```

Comment:
Note on compatibilty: it is compatible (I hope) in the interface part but we know nothing about compatibility at the storage layer.